kernel/sched/fair.c

```diff
@@ -6174,9 +6172,11 @@ static int do_sched_cfs_period_timer(struct cfs_bandwidth *cfs_b, int overrun, u
 
 		if (cfs_b->curr_interval < 10) {
 			cfs_b->recommender_active = false;
+			if (throttled && cfs_b->curr_interval > 1)
+				cfs_b->max_ratio = min((unsigned long long)num_online_cpus() * 100000, cfs_b->max_ratio * 2);
```

Comment:
do we need to reset max_ratio after the scale up is done and throttle is mitigated?

Reply:
I intended to do this. In this case, when next burst happens, it can satisfy the burst quickly.